courses/free-courses-it.md

```diff
@@ -170,6 +172,7 @@
 
 ### Programmazione
 
+* [Corso di C per sistemi Embedded](https://www.youtube.com/playlist?list=PLFPNVUgYSJN3szCCDYzDoai2ueDxZpCcb) - Davide Ferrero (Deid Lab)
```

Comment:
:information_source: I think the instructor was forgotten. Give credits :wink: 

Instructor: Nicolò Santilio

```markdown
* [Corso di C per sistemi Embedded](https://www.youtube.com/playlist?list=PLFPNVUgYSJN3szCCDYzDoai2ueDxZpCcb) - Nicolò Santilio
```

Reply:
> 
> 
> ℹ️ I think the instructor was forgotten. Give credits 😉
> 
> Instructor: Nicolò Santilio
> 
> ```gfm
> * [Corso di C per sistemi Embedded](https://www.youtube.com/playlist?list=PLFPNVUgYSJN3szCCDYzDoai2ueDxZpCcb) - Nicolò Santilio
> ```

The Credits are correct actually - Nicolò Santilio it's me, but the instructor is Davide Ferrero. Just created the playlist for that course and gave him the credits.